src/helper.js

```diff
@@ -29,4 +29,16 @@ export default class DistrictRepository {
       return undefined;
     }
   };
+
```

Comment:
i think that's a great idea. for now it's working so let's move on to visualization